go.mod

```diff
@@ -1,4 +1,6 @@
-module github.com/ravendb/ravendb-go-client
+module github.com/omerlewitz/ravendb-go-client
```

Comment:
No

Reply:
My remote was a mess.
I reset hard to raven-go-client and the only change I see atm is `	github.com/google/uuid v1.3.0` due to `raft id`
at the bottom of the PR page, I see that there's no mention my remote.